tools/download_paks.py

```diff
@@ -12,17 +12,16 @@
 
 # list all available scene_ids for download
 available_scene_ids = [
+    "Common",
     "kujiale_0000",
     "kujiale_0001",
     "kujiale_0002",
     "kujiale_0003",
     "kujiale_0004",
-    "kujiale_0005",
```

Comment:
Let's do comments rather than removing entirely. It will hint to the reader that the missing scenes will be back soon.